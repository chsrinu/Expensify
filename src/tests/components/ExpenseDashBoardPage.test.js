import React from 'react';
import {shallow } from 'enzyme';
import ExpenseDashBoardPage from '../../components/ExpenseDashBoardPage';

test('ExpenseDashBoardPage testCase',() => {
    const wrapper = shallow(<ExpenseDashBoardPage />);
    expect(wrapper).toMatchSnapshot();
})