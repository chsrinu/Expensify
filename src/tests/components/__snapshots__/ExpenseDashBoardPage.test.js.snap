// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ExpenseDashBoardPage testCase 1`] = `<Connect(ExpenseList) />`;
